docs/create-packages/Sign-a-Package.md

```diff
@@ -0,0 +1,101 @@
+---
+title: How to Create a Localized NuGet Package | Microsoft Docs
+author: rido-min
+ms.author: rido-min
+manager: unniravindranathan
+ms.date: 02/23/2018
+ms.topic: article
+ms.prod: nuget
+ms.technology: null
+description: Explains how signed packages can be used to enable content integrity verification
+keywords: NuGet package signing, NuGet security, creating signed packages
+ms.reviewer:
+- karann-msft
+- anangaur
+---
+
+# Signed packages
+
+NuGet packages can include a digital signature which provides protection against tampered content. This signature is produced from an x509 certificate that also adds authenticity proofs to the actual origin of the package. 
```

Comment:
Is there a specific version requirement for NuGet? Unless it applies all the way back to NuGet 2.8 (the earliest that's available from nuget.org/downloads), then we need to identify applicable versions.

From later on it looks like it's NuGet 4.6, correct?

Reply:
yes